packages/core/web-reporter-ui/src/components/Charts/Chart.tsx

```diff
@@ -19,7 +19,7 @@ const BaseChart = ({
 }: {
   type: Exclude<ApexChartProps["type"], undefined>;
   title: string;
-  series: LineSeriesType;
+  series: ApexAxisChartSeries;
```

Comment:
Just a doubt, where is this coming from? I don't see it in the imports

Reply:
It comes from apexcharts.d.ts which has `declare`d the type without exporting it... I didn't even realized it.